src/appleseed.studio/mainwindow/rendering/renderwidget.cpp

```diff
@@ -131,6 +131,13 @@ namespace
     }
 }
 
+void RenderWidget::start_render()
+{
+    // Clear the image storage.
+    if (m_image_storage.get())
```

Comment:
If you do a render, stop it, change something and re-render, if you change the display transform while rendering, old tiles from render # 1 appear on the render widget.